### ISSUE TITLE
Equivalent of torch.nn.functional.logsigmoid in dynet

### ISSUE THREAD

**Issue by Aditi138** [clab/dynet#1217]
Is there an equivalent function for torch.nn.functional.logsigmoid in dynet?
I tried the operation with dy.log(dy.sigmoid()) but that leads to nan's

**Comment by duyvuleo** [comment]
I think there is no log_sigmoid at the moment but you may do the equivalent
thing to avoid nan:

dy.log(dy.sigmoid(x) + sigma) where sigma = 10^-6 or smaller.

Not sure this is a right way.

Sent from Note 3

On Feb 4, 2018 12:17 PM, "Aditi" <notifications@github.com> wrote:

Is there an equivalent function for torch.nn.functional.logsigmoid in dynet?
I tried the operation with dy.log(dy.sigmoid()) but that leads to nan's

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/clab/dynet/issues/1217>, or mute the thread

**Comment by Aditi138** [comment]
Thanks! This helped.

**Comment by neubig** [comment]
This would be worth having as an actual function, so I'll leave this open.

**Comment by neubig** [comment]
Fixed by #1224
